### DISCUSSION TITLE
Production recommandations

### DISCUSSION THREAD

**Discussion by malves** [wevm/vocs#115] [General]
Hello all ! 

Thank you for your amazing work!  

I am not familiar with this stack. Do you have any recommandations or instructions on how to deploy my documentation to production on my own server ? I couldn't find any section about this in your documentation.

Thank you !

**Comment by codingwithmanny** [comment]
Technically when you run `pnpm build` it should create a `dist` folder with standard html files.

From there you should just be able to use any http server to support the files and folders.
